compiler/src/utils/config.re

```diff
@@ -306,8 +306,25 @@ let option_conv = ((prsr, prntr)) => (
     | Some(x) => prntr(ppf, x),
 );
 
-let optimizations_enabled =
-  toggle_flag(~doc="Disable optimizations.", ~names=["O0"], true);
+type optimization_level =
```

Comment:
I went pretty plain for this, but I could see us instead doing something like:
```
| Optimizations_disabled
| Critical_Grain_optimizations
| Grain_optimizations
| Grain_and_Binaryen_optimizations
```

but I felt pretty unsure about doing it that way.

Reply:
I think this is good, especially since it's more meant for compiler developers than end-users.